coordinates/sites.json

```
@@ -793,5 +793,91 @@
 	"longitude_unit": "degree",
 	"timezone": "US/Mountain",
 	"source": "NSO website: https://nsosp-dev.nso.edu/node/18"
+    },
+    "teide": {
+        "name": "Observatorio del Teide, Tenerife",
+        "elevation": 2390,
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 28.3,
+        "elevation_unit": "meter",
+        "longitude": 343.490278,
+        "timezone": "Atlantic/Canary",
+        "source": "Instituto de Astrofisica de Canarias website (http://research.iac.es/OOCC/observatorio-del-teide/ot/)",
+        "aliases": [
+            "Observatorio del Teide",
+            "OT"
+        ]
+    },
+    "CAHA": {
+        "name": "Centro Astronomico Hispano-Aleman, Almeria",
+        "elevation": 2168,
+        "longitude_unit": "degree",
+        "latitude_unit": "degree",
+        "latitude": 37.22361111,
+        "elevation_unit": "meter",
+        "longitude": 357.453889,
+        "timezone": "CET",
```

```suggestion
        "timezone": "Europe/Madrid",
```
Again, with the discussion in https://github.com/astropy/astropy-data/issues/117#issuecomment-922374617 and `US/Hawaii`, `Atlantic/Canary` above I vote for entering all TZ consistently, so DST information is unambiguous wherever applicable.